### PR TITLE
Portal editor to same container used to portal other components

### PR DESCRIPTION
**Pull request by saskliutas**
## Changes

Container used to portal components had higher z-index than `Popup` components. Changed editors to use same portal container. Required to fix https://github.com/iTwin/viewer-components-react/issues/921.

## Testing

Tested manually

### CODE REVIEW

File: common/api/core-react.api.md [hunk 1]
```diff
@@ -1230,6 +1230,7 @@ export interface PopupProps extends CommonProps {
     onOpen?: () => void;
     onOutsideClick?: (e: MouseEvent) => void;
     onWheel?: (e: WheelEvent) => void;
+    portalTarget?: HTMLElement;
```
Comment (GerardasB) [inline common/api/core-react.api.md:1233]: API addition, document in NextVersion

**Review by GerardasB — APPROVED** [review]

File: common/api/core-react.api.md (hunk 1 above)
Comment (saskliutas) [reply common/api/core-react.api.md:1233]: Should be advertise changes to deprecated components?

Comment (GerardasB) [reply common/api/core-react.api.md:1233]: I don't think it's an issue. Changelogs should be complete with all the changes to avoid any surprises.

Comment (saskliutas) [reply common/api/core-react.api.md:1233]: Updated changelog

**Pull request merged**